WORKSPACE

```diff
@@ -5,26 +5,23 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # GoogleTest/GoogleMock framework. Used by most unit-tests.
 http_archive(
     name = "com_google_googletest",
-    sha256 = "94c634d499558a76fa649edb13721dce6e98fb1e7018dfaeba3cd7a083945e91",
-    strip_prefix = "googletest-release-1.10.0",
-    urls = ["https://github.com/google/googletest/archive/release-1.10.0.zip"],
+    sha256 = "353571c2440176ded91c2de6d6cd88ddd41401d14692ec1f99e35d013feda55a",
+    strip_prefix = "googletest-release-1.11.0",
+    urls = ["https://github.com/google/googletest/archive/release-1.11.0.zip"],
 )
 
 # Google Benchmark library.
 http_archive(
     name = "com_github_google_benchmark",
-    sha256 = "3e266b49f73ee08625837ea5b1fabc4056b7f5e809b29c49670527326f4f4379",
-    strip_prefix = "benchmark-1.5.1",
-    urls = ["https://github.com/google/benchmark/archive/v1.5.1.zip"],
+    sha256 = "30f2e5156de241789d772dd8b130c1cb5d33473cc2f29e4008eab680df7bd1f0",
+    strip_prefix = "benchmark-1.5.5",
+    urls = ["https://github.com/google/benchmark/archive/v1.5.5.zip"],
 )
 
 # C++ rules for Bazel.
 http_archive(
     name = "rules_cc",
-    sha256 = "fa42eade3cad9190c2a6286a6213f07f1a83d26d9f082d56f526d014c6ea7444",
-    strip_prefix = "rules_cc-02becfef8bc97bda4f9bb64e153f1b0671aec4ba",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/rules_cc/archive/02becfef8bc97bda4f9bb64e153f1b0671aec4ba.zip",
-        "https://github.com/bazelbuild/rules_cc/archive/02becfef8bc97bda4f9bb64e153f1b0671aec4ba.zip",
-    ],
+    sha256 = "f3908cb40a6577ab0d1ef9e00052739bf69b4313fa7d20b4d61d4521ea67abf3",
+    strip_prefix = "rules_cc-ab5395627c80e025e824bd005d41f96b20618b9d",
+    urls = ["https://github.com/bazelbuild/rules_cc/archive/ab5395627c80e025e824bd005d41f96b20618b9d.zip"],
```

Comment:
I used the current head (daf6ace7cfeacd6a83e9ff2ed659f416537b6c74) for this one.  Any reason not to?

Reply:
I actually just reverted this. It looks like we need to update the Bazel version of Windows/Mac before we can update this.

Reply:
Ack.